### PR TITLE
feat(TMST-623): create article card

### PR DESCRIPTION
**Pull request by josipVuko**
[TMST-623](https://nidigitalsolutions.jira.com/browse/TMST-623)

Description

- Create RecentStory card component using the NK components
- Wrote unit test for RecentStory card component
- Wrote story for  RecentStory card component

image

[TMST-623]: https://nidigitalsolutions.jira.com/browse/TMST-623?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/ts-newskit/src/components/puzzles/storyCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+              marginBlock: 'space040',
+              typographyPreset: 'editorialHeadline020'
+            }}
+          >
+            {title}
+          </Headline>
+        </CardLink>
+      </CardContent>
+      <CardActions>
+        <GridLayout columns="repeat(2, auto)" columnGap="space020">
```
Comment (jasminaj0) [inline packages/ts-newskit/src/components/puzzles/storyCard/index.tsx:74]: In cases like this, you can also use `Stack `component that can take horizontal or vertical flow. Is there a reason why you used `GridLayout`?

File: packages/ts-newskit/src/components/puzzles/storyCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  Tag,
+  CardComposable,
+  Visible,
+  CardLink,
+  Headline,
+  CardMedia,
+  GridLayout
+} from 'newskit';
+
+export interface StoryCardProps {
```
Comment (jasminaj0) [inline packages/ts-newskit/src/components/puzzles/storyCard/index.tsx:15]: I would suggest to create and use types according to the TPA schema, and if any filed is missing, to add it as a comment, we need to track things we will need when it comes to extending schema.

File: packages/ts-newskit/src/components/puzzles/storyCard/StoryCard.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Code Example
+`<StoryCard data={image, altText, title, url, category, timeToRead, imgHiddenMobile} />`
+
+## View Component
+Please click the 'Canvas' tab for a better viewing experience, where you can update the props and review at the different breakpoints by clicking the preview icon and selecting from our list of pre-defined breakpoints (XS, SM, MD, LG and XL).
+
+export const StoryCardStory = ({ image, altText, title, url, category, timeToRead, imgHiddenMobile }) => (
+  <TCThemeProvider>
+    <StoryCard {...{ image, altText, title, url, category, timeToRead, imgHiddenMobile}} />
```
[the remark this answers is not in the document]
Comment (josipVu) [reply packages/ts-newskit/src/components/puzzles/storyCard/StoryCard.stories.mdx]: Added composable card to keep fixed width on desktop and tablet.

File: packages/ts-newskit/src/components/puzzles/storyCard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+          <NewsKitArticlePlaceholder data-testid="storyCard-placeholder" />
+        )}
+      </Visible>
+      <CardContent>
+        <CardLink
+          expand
+          href={url}
+          data-testid="storyCard-link"
+          overrides={{
+            externalIcon: {
```
[the remark this answers is not in the document]
Comment (josipVu) [reply packages/ts-newskit/src/components/puzzles/storyCard/index.tsx:56]: I get icon displayed that way so I will keep "size:0" property.

File: packages/ts-newskit/src/theme/times-web-light/style-presets/storyCard.ts [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    }
+  },
+  StoryCardTime: {
+    base: {
+      color: '{{colors.inkSubtle}}',
+      border: 'none',
+      padding: '0'
+    }
+  },
+  StoryCardFlexWrapper: {
```
Comment (ibrahim-kurhan) [inline packages/ts-newskit/src/theme/times-web-light/style-presets/storyCard.ts:16]: is this StoryCardFlexWrapper being used anywhere?

**Review by jasminaj0 — APPROVED** [review]

**Pull request merged**